samples/system-test/importProductSets.v1p3beta1.test.js

```diff
@@ -19,13 +19,13 @@ const path = require(`path`);
 const assert = require('assert');
 const tools = require(`@google-cloud/nodejs-repo-tools`);
 const cmd = `node importProductSets.v1p3beta1.js`;
-const cwd = path.join(__dirname, `..`);
+const cwd = path.join(__dirname, `..`, `productSearch`);
 
 //Shared fixture data for product tests
 const testImportProductSets = {
   projectId: process.env.GCLOUD_PROJECT,
   location: 'us-west1',
-  gcsUri: 'gs://nodejs-docs-samples/product-search/product_sets.csv',
+  gcsUri: 'gs://product-search-node/product_sets.csv',
```

Comment:
What project does this bucket belong to? We try to keep all of our test resources in a single project for testing.

Reply:
Fixed